charts/super-agent-deployment/README.md

```diff
@@ -0,0 +1,85 @@
+means that it honors a wide range of defaults and globals common to most New Relic Helm charts.
+
+Options that can be defined globally include `affinity`, `nodeSelector`, `tolerations`, `proxy` and others. The full list can be found at
+[user's guide of the common library](https://github.com/newrelic/helm-charts/blob/master/library/common-library/README.md).
+
+## Chart particularities
+
+> **TODO:** Here is where you should add particularities for this chart like what does the chart do with the privileged and
+low data modes or any other quirk that it could have.
+
+### TODOs
+There are values that should be planned at some point of removed from the `values.yaml`. I leave them here documented
+as TODO:
+ * `licenseKey`: comes from the common library but it would not be needed as it is an ingestion API Key, not a REST one.
+ * `rbac`: the is a placeholder for RBAC that simply list pods. this has to be narrowed to the use case of this agent.
+ * `customAttributes`: decorate everything with this custom attributes, maybe as they come from opamp.
+ * `proxy`, `nrStaging` and `fedramp` support on the meta agent. This could be made from the chart itself changing the opamp endpoint.
+ * `verboseLog`: legacy toggle to enable verbosity.
+
+Other TODOs:
+ * Add probes for liveness and readiness.
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| affinity | object | `{}` | Sets pod/node affinities. Can be configured also with `global.affinity` |
+| cluster | string | `""` | Name of the Kubernetes cluster monitored. Can be configured also with `global.cluster`. |
+| config | object | See `values.yaml` for examples | Here you can set New Relic' Super Agent configuration. |
+| containerSecurityContext | object | `{}` | Sets security context (at container level). Can be configured also with `global.containerSecurityContext` |
+| customAttributes | object | `{}` | Adds extra attributes to the cluster and all the metrics emitted to the backend. Can be configured also with `global.customAttributes` |
+| customSecretLicenseKey | string | `""` | In case you don't want to have the license key in you values, this allows you to point to which secret key is the license key located. Can be configured also with `global.customSecretLicenseKey` |
+| customSecretName | string | `""` | In case you don't want to have the license key in you values, this allows you to point to a user created secret to get the key from there. Can be configured also with `global.customSecretName` |
+| dnsConfig | object | `{}` | Sets pod's dnsConfig. Can be configured also with `global.dnsConfig` |
+| extraVolumeMounts | list | `[]` | Defines where to mount volumes specified with `extraVolumes` |
+| extraVolumes | list | `[]` | Volumes to mount in the containers |
+| fedramp.enabled | bool | `false` | Enables FedRAMP. Can be configured also with `global.fedramp.enabled` |
```

Comment:
I would not include in the docs features not implemented yet

Reply:
It was marked as TODO here: https://github.com/newrelic/helm-charts/pull/1155#discussion_r1365584627